docs/index.md

```diff
@@ -1,4 +1,4 @@
-# BitcoinSource v0.0.1-dev
+# BitcoinSource vv0.1.12
```

Comment:
Remove vv?

Reply:
Got it. Markdown needs one 'v' to pass the test. Let's discuss the need for this test later.